src/core/libraries/Helpers.sol

```diff
@@ -201,11 +210,20 @@ library Helpers {
                 revert Errors.SablierLockup_CliffTimeNotLessThanEndTime(cliffTime, timestamps.end);
             }
         }
+        // Check: the cliff unlock amount is zero when the cliff time is zero.
+        else if (unlockAmounts.cliff > 0) {
+            revert Errors.SablierLockup_CliffUnlockAmountNotZero(unlockAmounts.cliff);
```

Comment:
The error name is unclear. How about `SablierLockup_CliffTimeZeroUnlockAmountNotZero`?